src/classes/frDonation.cls

```diff
@@ -77,10 +77,12 @@ public class frDonation extends frModel {
         this.setDonorId(String.valueOf(request.get('donorId')));
         String status = String.valueOf(request.get('status'));
         
-        if (Boolean.valueOf(request.get('pledge'))) {
```

Comment:
Was there a reason this was checking if the donation was a pledge? Are we losing anything by changing this check to if the stage name is empty?

Reply:
I don't believe we are losing anything.  Previously the behavior would be we would only set the opportunity to Pending if it was BOTH a pledge and in Pending status in Funraise.  

With these changes, we'll now set it to Pending in Salesforce regardless if it's a pledge or not in Funraise, as long as the status is Pending in Funraise, which I think makes sense for the statuses to match up regardless of being a pledge or not (though right now this may not work out to much functional changes since I don't think we send any pending donations that ARENT pledges)